system/settings/settings.xml

```diff
@@ -1411,7 +1411,7 @@
         </setting>
         <setting id="pvrplayback.confirmchannelswitch" type="boolean" label="19281" help="36231">
           <level>1</level>
-          <default>false</default>
+          <default>true</default>
```

Comment:
I don't like this because this setting require a confirmation for switching channel with numeric input. If it is changed by this PR, I don't mind